aidial_adapter_openai/utils/tokens.py

```diff
@@ -43,62 +44,23 @@ def calculate_tokens_per_message(self, message: dict) -> int:
 
         tokens = tokens_per_message
         for key, value in message.items():
-            if isinstance(value, str):
-                tokens += self.calculate_tokens(value)
             if key == "name":
                 tokens += tokens_per_name
-
+            elif isinstance(value, str):
```

Comment:
Maybe this structure? It looks much more straight-forward:

```python
if name == "name": ...
elif name == "content":
    if isinstance(value, str): ....
    elif isinstance(value, list): ....
```

Reply:
Why do we need to introduce this extra nesting?

Content with string in value will be processed by 
```python
elif isinstance(value, str):
```
as any other key in message

with this `if` I want to cover one case, and one case only - when content has custom content in it